### DISCUSSION TITLE
Hacking Attempt - Cause for Concern?  System Compromised?

### DISCUSSION THREAD

**Discussion by cupanther** [GerryDazoo/Slinger#483] [Q&A] [answered]
I mainly use slinger for viewing outside of my home network.  I am running version 3.08e on Windows 10 and got the following alert:

Hacker Alert. Invalid Request from  ('94.102.61.39', 37400)
GET / HTTP/1.1
Host: [my.external.ip:port](http://[50.36.25.104:8097](my.external.ip:port)/)
User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4324.190 Safari/537.36
Accept: */*
Accept-Encoding: gzip

The requesting IP appears to be from the Netherlands and not me.  What does the rest of the information mean?  Was something deployed to my system?

**Comment by GerryDazoo** [comment, marked as answer]
First. You should be running the latest code.
Nothing was deployed to your system. It's a dump of the http Get header sent by buddy in the Netherlands.
